docs/source/collect/run-pod.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Run Pods
+description: Running pods during collection time to create data
+---
+
+> Looking for the old Run collector? See: [Run (Deprecated)](https://troubleshoot.sh/docs/collect/deprecated/run)
+
+The `runPod` collector can be used to run a pod in the cluster with the parameters provided.
+The collector will delete and clean up this pod and any artifacts after it's created.
+This collector can be included multiple times, each defining different commands to run.
+
+## Parameters
+
+In addition to the [shared collector properties](https://troubleshoot.sh/docs/collect/collectors/#shared-properties), the `runPod` collector accepts the following parameters:
+
+##### `name` (Optional)
+The name of the collector. This will be prefixed to the path that the output is written to in the support bundle.
+
+##### `namespace` (Optional)
+The namespace to look for the pod selector in.
+If not specified, it will assume the "current" namespace that the kubectl context is set to.
+
+##### `podSpec` (Required)
+
+> Introduced in Troubleshoot v0.33.0.
+
+The `corev1.PodSpec` for the `runPod` collector. See the [Kubernetes API Reference](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.23/#podspec-v1-core) for all available properties.
+
+##### `timeout` (Optional)
+A [duration](https://golang.org/pkg/time/#Duration) that will be honored when running the pod.
+This cannot be greater than 30 seconds (30s) and if not specified, the default is 20s.
+
+#### `imagePullSecret` (Optional)
+
+> `imagePullSecret` support was introduced in Kots 1.19.0 and Troubleshoot 0.9.42.
```

Comment:
im confused. if the runPod collector was introduced in 0.33.0 how can `imagePullSecret` be introduced before then?

Reply:
Good catch. No need to keep that part in the new collector doc